imod/mf6/simulation.py

```diff
@@ -1248,4 +1249,37 @@ def _update_buoyancy_packages(self)->None:
         flow_transport_mapping = self._get_transport_models_per_flow_model()
         for flow_name, tpt_models_of_flow_model in flow_transport_mapping.items():
             flow_model = self[flow_name]
-            flow_model.update_buoyancy_package(tpt_models_of_flow_model)
\ No newline at end of file
+            flow_model.update_buoyancy_package(tpt_models_of_flow_model)
+
+
+    def mask_all_models(
+        self,
+        mask: GridDataArray,
+    ):
+        """
+        This function applies a mask to all models in a simulation, provided they use
+        the same discretization. 
+
+        Parameters
+        ----------
+        mask: xr.DataArray, xu.UgridDataArray of ints
+            idomain-like integer array. 1 sets cells to active, 0 sets cells to inactive, 
```

Comment:
This is basically the definition of the idomain. Are you masking the original domain or are you defining a new idomain? A method name like `alter_cell_passthrough` would also describe this method